parsely_raw_data/schema.py

```diff
@@ -26,6 +26,7 @@
     {"key": "action", "ex": "pageview", "type": str, "size": 256, "req": True},
     {"key": "apikey", "ex": "mashable.com", "type": str, "size": 256, "req": True},
     {"key": "campaign_id", "ex": "facebook_campaign", "type": str, "size": 256, "available_with_field": "url_query"},
+    {"key": "channel", "ex": "AMP", "type": str, "size": 64},
```

Comment:
nitpick: this will be lowercase, as per internal code (i.e. `amp`).

Reply:
+1 , I was looking at Dash